packages/daemon/src/directory.js

```diff
@@ -126,16 +148,16 @@ export const makeDirectoryMaker = ({
       return harden(Array.from(identities).sort());
     };
 
-    /** @type {import('./types.js').EndoDirectory['followChanges']} */
-    const followChanges = async function* followChanges(...petNamePath) {
+    /** @type {import('./types.js').EndoDirectory['followNames']} */
+    const followNames = async function* followNames(...petNamePath) {
```

Comment:
I like this change, but will note that @kumavis made the opposite change, so we should get on the same page. I suppose @kumavis liked having `Changes` in the name because `followNames` implies snapshots. I could meet up at `followNameChanges`, to leave room for the other form.

Reply:
While pretty clear about the format, I find `followChanges` too non-specific regarding _which_ changes are being followed. I'd be okay with `followNameChanges` / `followLocatorNameChanges`, but I also think it's fine to force consumers to look at the docs to figure out the format and go with the terser `followNames` / `followLocatorNames`. Personally, I wouldn't expect anything about the format with the latter formulation.

I've pinged @kumavis for feedback, and we can await his word to form a consensus.

Reply:
I like having "changes" when the iterated entries are diffs rather than values

Reply:
> but I also think it's fine to force consumers to look at the docs to figure out the format

To be clear: I think this because consumers will have to look at the docs anyway. There is more than one way to format both snapshots and incremental changes.

Reply:
@kriskowal wanna break the tie?

Reply:
Let’s run with `followNameChanges`.